### DISCUSSION TITLE
Persist Video Location When Entering Post

### DISCUSSION THREAD

**Discussion by patrickdemers6** [Docile-Alligator/Infinity-For-Reddit#1158] [Ideas]
# Scenario
- You are scrolling through your feed and come across an interesting video.
- You watch 15 seconds of it.
- You click into the post.

# Current Behavior
When you click into the post, the video starts from the beginning.

# Possible Improvement
When you click into the post, the video continues where you left off.

**Comment by Wladefant** [comment]
Thank you (nice structure)
Fully agree.
